rpcserver.go

```diff
@@ -258,6 +258,51 @@ func calculateFeeRate(satPerByte, satPerVByte uint64, targetConf uint32,
 
 }
 
+// GetAllPermissions returns all the permissions required to interact with lnd.
+func GetAllPermissions() []bakery.Op {
+	allPerms := make([]bakery.Op, 0)
+	// The map will help keep track of which specific permission pairs have
+	// already been added to the slice.
+	allPermsMap := make(map[string][]string)
```

Comment:
This could be simplified by a double map (`map[string]map[string]struct{}`) that just map `entity -> action`. That way you could just insert into the map and in the end loop over it again to get de-duplicated values.